File: src/main/java/no/nav/melosys/eessi/models/sed/nav/Grunnlag.java
package no.nav.melosys.eessi.models.sed.nav;

import com.fasterxml.jackson.annotation.JsonInclude;
import com.fasterxml.jackson.annotation.JsonInclude.Include;
import lombok.Data;

@SuppressWarnings("unused")
@JsonInclude(Include.NON_NULL)
@Data
public class Grunnlag {

    private String annet;

    private String artikkel;
}

File: melosys-eessi-app/src/main/java/no/nav/melosys/eessi/models/sed/nav/Grunnlag.kt
package no.nav.melosys.eessi.models.sed.nav

import com.fasterxml.jackson.annotation.JsonInclude

@JsonInclude(JsonInclude.Include.NON_NULL)
data class Grunnlag (
    var annet: String? = null,
    var artikkel: String? = null
)
